airbyte-integrations/connectors/source-sftp-bulk/source_sftp_bulk/client.py

```diff
@@ -135,11 +150,22 @@ def is_directory(a):
                         "last_modified": datetime.utcfromtimestamp(last_modified).replace(tzinfo=None),
                     }
                 )
-
         return files
 
-    def get_files(self, prefix, search_pattern=None, modified_since=None, most_recent_only=False) -> List[File]:
-        files = self.get_files_by_prefix(prefix)
+    def get_files(
+        self,
+        prefix,
+        search_pattern=None,
+        modified_since=None,
+        most_recent_only=False,
+        max_level=0,
```

Comment:
This means it will change the current behavior or the connector to only root level, right?

Reply:
yes, it's the existing behaviour